packages/squiggle-lang/__tests__/TS/Parser_test.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  run,
+  squiggleExpression,
+  errorValue,
+  result,
+} from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Squiggle is whitespace insensitive", () => {
+  test("when assigning a distribution to a name and calling that name", () => {
+    /*
+     * intersperse varying amounts of whitespace in a squiggle string
```

Comment:
Why not a one-line comment?

Reply:
no reason.

Reply:
I could see that, but would probably still prefer it to be one-line, if its' not being used as a docstring.